docs/hermes.md

```diff
@@ -52,3 +52,25 @@ $ react-native run-android --variant release
 ```
 
 This will compile JavaScript to bytecode during build time which will improve your app's startup speed on device.
+
+## Debugging Hermes using the Chrome Inspector
```

Comment:
Would this be better as "Debugging Hermes using the Chrome DevTools"?

Reply:
Sounds good. I didn't really know what terminology people use here.